mscore/musescore.cpp

```diff
@@ -5754,6 +5770,50 @@ void MuseScore::transpose()
             cs->deselectAll();
       }
 
+//---------------------------------------------------------
+//   cmdRealizeChordSymbols
+///   Realize selected chord symbols into notes on the staff.
+///   Currently just pops up a dialog to list TPCs,
+///   Intervals, and pitches.
+//---------------------------------------------------------
+
+void MuseScore::realizeChordSymbols()
+      {
+      if (!cs)
+            return;
+      if (!cs->selection().isList()) {
+            QErrorMessage err;
+            err.showMessage("Invalid Selection. Cannot Realize Chord");
```

Comment:
(probably other places throughout) - be sure to enclosed string literals displayed to user in tr("") or similar macro (see how it is handled elsewhere).

I noticed this first with the channel name, but that's correct *not* to translate.  Again, check how other channel names are handled.

Reply:
here though it is indeed definitly needed